BalancedBinaryTree.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Definition for a binary tree node.
+ * public class TreeNode {
+ *     int val;
+ *     TreeNode left;
+ *     TreeNode right;
+ *     TreeNode() {}
+ *     TreeNode(int val) { this.val = val; }
+ *     TreeNode(int val, TreeNode left, TreeNode right) {
+ *         this.val = val;
+ *         this.left = left;
+ *         this.right = right;
+ *     }
+ * }
+ */
+class Solution {
+
+    // Time Complexity : 0(n)
+// Space Complexity : o(h)
```

Comment:
Please specify what `n`  and `h` indicates.

Reply:
n is the nodes and h is the height of the binary tree